utils/obsproc/Smos2Ioda.h

```diff
@@ -80,7 +80,7 @@ namespace gdasapp {
         iodaVars.obsVal_(i) = sss[i];
         iodaVars.obsError_(i) = sss_error[i];
         iodaVars.preQc_(i) = sss_qc[i];
-        iodaVars.datetime_(i) =  static_cast<int64_t>(datetime_[i]*86400.0f) + mjd2000;
+        iodaVars.datetime_(i) =  static_cast<int64_t>((datetime_[i]*86400.0f) + mjd2000);
```

Comment:
is there a point to the addition of the parenthesis?

Reply:
No, it is correct and I reverse smos.